x-pack/test/api_integration/apis/maps/proxy_api.js

```diff
@@ -22,7 +22,7 @@ export default function ({ getService }) {
       //Check world-layer
       const worldLayer = resp.body.layers.find((layer) => layer.layer_id === 'world_countries');
       expect(worldLayer.formats.length).to.be.greaterThan(0);
-      expect(worldLayer.formats[0].type).to.be('geojson');
+      expect(worldLayer.formats[0].type).to.be('topojson');
```

Comment:
How did https://github.com/elastic/kibana/pull/79760 originally pass? Does the order of `formats` change?

Reply:
this is because we published an addition to EMS this afternoon, now including the topojson format 7.10 and upward. This format was _not_ included before.